### DISCUSSION TITLE
data attribute

### DISCUSSION THREAD

**Discussion by jamesgrubb** [alpinejs/alpine#940] [1. Help] [answered]
Him, I have a `data-src` attribute that pulls in an SVG is there a way of accessing that like you can with `:src` for example?
```
 <svg x-show="loading" class="h-4"
    data-src="https://s.svgbox.net/loaders.svg?ic=spinner"
    fill="currentColor"></svg>
```

Thanks

**Comment by SimoTod** [comment, marked as answer]
You can add something like `x-ref="mysvg"` and when you need to read the value you can use `$refs.mysvg.dataset.src`

`mysvg` is just a name and you can call it whatever you want.

**Reply by jamesgrubb** [reply]
Thanks, sorry for the late reply
